.circleci/config.yml

```diff
@@ -41,6 +41,19 @@ jobs:
       - checkout
       - install-dependencies
       - build-site
+      - persist_to_workspace:
+          root: "."
+          paths:
+            - "_site"
```

Comment:
> do we want to persist the `node_modules` etc too? or do we assume that caching will take care of that for us

I did that at first, and I found that it took a very long time for the `persist_to_workspace` step itself to complete, probably because `node_modules` is a block hole. And yeah, was hopeful that the caching in `install-dependencies` could keep it fast enough.

See: fe78bdf1fc614878c1eff9d34df5932b082f94ec